ruby_event_store/lib/ruby_event_store/mappers/default.rb

```diff
@@ -3,6 +3,8 @@
 module RubyEventStore
   module Mappers
     class Default
+      include PipelineMapper
```

Comment:
Maybe that's already in next commits: doesn't feel right to include PipelineMapper and depend on a presence of an ivar.

Either `include PipelineMapper.new(...)` with all pipeline stuff or `Default = PipelineMapper.new(...)`.

Reply:
Would be great but it will break backward compatibility - we need some arguments given in mapper initializer to build a pipeline.

Reply:
OK, no breaking compatibility. I'd however revert that `include PipelineMapper` module. Having mapper methods explicitly delegate to pipeline is fine composition. I know this basically happens in the included module but it is not instantly visible. 

What do we gain with that extraction?